dotcom-rendering/src/web/lib/verticalDividerWithBottomOffset.ts

```diff
@@ -0,0 +1,24 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { border, from } from '@guardian/source-foundations';
+
+export function verticalDividerWithBottomOffset(
```

Comment:
Do we have plans for this to be used in other places? If not would it be better to live in LI.tsx?